### PR TITLE
(doc) Add documentation about caching functionality

### PR DESCRIPTION
**Pull request by AdmiringWorm**
## Description Of Changes

This PR adds a new features section with public information
about the new HTTP caching solution that was introduced with
Chocolatey CLI v2.0.0.

## Motivation and Context

To give some information about new functionality in Chocolatey CLI.

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [ ] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [x] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.

## Change Checklist

* [x] Requires a change to menu structure (top or left hand side)/
* [x] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

File: input/en-us/choco/features/http-caching.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Any created HTTP cache will be valid for 30 minutes. After this time has passed, any new requests made for the cached items will reach out to the appropriate feeds again and replace the expired cache with the new response.
+
+It is important to note that Chocolatey CLI will not remove any caches, but rather a request to the cached call replaces it.
+
+### What To Do if Experiencing Problems With Caching / Force Up To Date Requests
+
+It is necessary to make manual changes if you are experiencing problems with the caching solution or want to force Chocolatey CLI to get the latest up-to-date information.
+
+It is required to delete the HTTP Cache directories manually. See [Locations for HTTP Cache](#locations-for-http-cache) for information about these locations.
```
Comment (vexx32) [inline input/en-us/choco/features/http-caching.md:46]: Phrasing here I think could be a little better. Maybe something like this?

```suggestion
If you are experiencing problems with the caching solution or want to force Chocolatey CLI to get the latest up-to-date information, you will need to delete the cached data manually. Refer to [Locations for HTTP Cache](#locations-for-http-cache) to find the appropriate cache location for your use case. Removing all files in these locations will clear the cached data and prompt Chocolatey CLI to reach out to the appropriate feeds when it next needs that data.
```

File: input/en-us/choco/features/http-caching.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+> :choco-info: **NOTE**
+>
+> We expect to add alternative methods to skip, ignore or clear the cache directories through Chocolatey CLI.
```
Comment (vexx32) [inline input/en-us/choco/features/http-caching.md:50]:
```suggestion
> We expect to add alternative methods to skip, ignore or clear the cache directories through Chocolatey CLI in a future version.

```

**Review by vexx32 — CHANGES_REQUESTED** [review]
Looks pretty good, just a couple minor suggestions 💖

File: input/en-us/choco/features/http-caching.md (hunk 1 above)
Comment (AdmiringWorm) [reply input/en-us/choco/features/http-caching.md]: I have updated this with a small change that reads clearer to me.

File: input/en-us/choco/features/http-caching.md (hunk 2 above)
Comment (AdmiringWorm) [reply input/en-us/choco/features/http-caching.md]: I was not too fond of that exact wording, but I have updated it to an alternative.

**Review by vexx32 — APPROVED** [review]
Looks good to me, great work! 💖

**Pull request merged**